blogWithComments.php

```diff
@@ -0,0 +1,90 @@
+
+<?php
+include ('include/include_all.php');
+
+session_start();
+if (!isset($_SESSION['UserId'])){
+	die("You're not logged in. <a href = 'http://localhost:8888/blogLogin.php'> Go to the login page </a>");
+}
+?>
+
+
+
+
+<html>
+
+	<head>
+		<title>Lindsay's Blog</title>
+		<link rel='stylesheet' href='maindesign.css' />
+		<link href="https://fonts.googleapis.com/css?family=Lobster|Montserrat" rel="stylesheet">
+	</head>
+
+
+	<body>
+		<h1 class='header'> Lindsay's Blog </h1>
+		<h2 class='subtitle'>WORDS FROM A GIRL LEARNING HOW TO CODE </h1>
+		<p class='logInNotice'>(You're logged in.  <a href ='http://localhost:8888/blogLogOut.php'> Click here to log out.</a>)</p>
+
+			<?php
+
+
+Block(4);
+Block(5);
+Block(6);
+
+
+
+echo "
+	<h1 class='SubmitAComment'> Submit a comment </h1>
+	<form action='' method='post'>
+		<table class='SubmitForm'>
+		  <tr>
+		    <td>
+		      Name:
+		    </td>
+		    <td>";
+					RepopulateComments('NameField');
+echo
+		    "</td>
+		  </tr>
+		  <tr>
+		    <td>
+		      Email:
+		    </td>
+		    <td>";
+					RepopulateComments('EmailField');
+echo
+				"</td>
+		  </tr>
+			<tr>
+				<td>
+					Comment:
+				</td>
+				<td>";
+					RepopulateComments('CommentField');
```

Comment:
You might want this to be a textarea instead of an input. That way people can leave multi-line comments